dinstaller-lib/share/profile.schema.json

```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/d-installer/profile.schema.json",
+  "title": "Profile",
+  "description": "Profile definition for automated installation",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "software": {
+      "description": "Software settings",
+      "type": "object",
+      "properties": {
+        "product": {
+          "description": "Product identifier",
+          "type": "string"
+        }
+      },
+      "required": [
```

Comment:
is it really required? what about ALP only medium?

Reply:
or customer medium with single product?

Reply:
Yes, true. We can relax this requirement, but we need to handle it at the code level.